features/forgot_password.feature

```diff
@@ -0,0 +1,16 @@
+Feature:
+  A working "Forgot-Password" mechanism
+
+  Scenario: A User starts the "Forgot-Password" Routine
+    Given a new user ulf with email ulf@ulf.de
+    And ulf is stored in the database
+
+    When ulf visits the password recovery page
```

Comment:
Consider using a background for this scenario, since this seems like two scenarios...

Reply:
:-1: